src/main/java/by/itech/lab/supplier/domain/Customer.java

```diff
@@ -38,6 +40,7 @@
     private boolean active;
     private LocalDate deletedAt;
     @OneToMany(mappedBy = "customer")
+    @EqualsAndHashCode.Exclude
```

Comment:
Can we put this  annotation on the class level? It's kind of metadata and it's easier to look at one place and understand what is excluded, otherwise you need to find it in the code.

Reply:
I think, we don't need this annotation at all